src/cljs/rems/roles.cljs

```diff
@@ -20,6 +20,9 @@
 (defn show-admin-edit-buttons? [roles]
   (some #{:organization-owner :owner} roles))
 
+(defn disallow-setting-organization? [roles]
+  (some #{:organization-owner} roles))
```

Comment:
should this also check that roles does _not_ contain :owner?

Reply:
Done.